app/utils/passwordEncrypt.ts

```diff
@@ -0,0 +1,12 @@
+import bcryptjs from 'bcryptjs';
```

Comment:
use snake_case to name the file https://github.com/Wolox/tech-guides/blob/master/nodejs/docs/node-standard-and-best-practices.md#31--files